.pre-commit-config.yaml

```diff
@@ -32,6 +32,10 @@ repos:
       - id: ruff
         types_or: [ python, pyi, jupyter ]
         args: ["--fix", "--ignore=E402"]
+        exclude: |
+            (?x)^(
+              notebook/agentchat_databricks_dbrx.ipynb
```

Comment:
This exclude didn't work as the ruff error still shows up in CI. Can you use full path rather than a regular expression?

Reply:
Ruff is being skipped in pre-commit now